### PR TITLE
Team showcasing on profile for desktop and mobile

### PR DESCRIPTION
**Pull request by cjb**
@keybase/react-hackers 

* The userInfo we receive as part of an identify now contains a showcasedTeams array, store it alongside the tracker info.
* When rendering the profile view, draw this list of teams.
* When we click on one, use the relative popup hoc (there are some changes to it from @MarcoPolo in this PR) to draw a relative positioned modal that shows the `showcased-team-info` component inside.  On mobile, just use a normal popup modal instead.
* When loading that component, pull out the team's description, whether it's open, number of members, whether we've already requested access to join.
* There's a button inside that component for joining the team.  If clicked we try to join and route the result through to the button, showing any error on the page.
* There's a new "publicAdmins" list for each team, with specific sorting requirements added to the container.  You can click on each username listed to load their profile.

### CODE REVIEW

File: shared/profile/showcased-team-info/container.js [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+    youAreInTeam,
+    youHaveRequestedAccess,
+  }
+}
+
+const mapDispatchToProps = (dispatch: Dispatch, {navigateUp, routeProps}) => {
+  const teamname = routeProps.get('team').fqName
+  return {
+    _checkRequestedAccess: (teamname: string) => dispatch(checkRequestedAccess(teamname)),
+    _loadTeam: (teamname: string) => dispatch(getDetails(teamname)),
```
Comment (chrisnojima) [inline shared/profile/showcased-team-info/container.js]: Why take a team name when you have it? Or do you expect a different one coming in?

Comment (cjb) [reply shared/profile/showcased-team-info/container.js]: Thanks, you're right, fixed.

File: shared/chat/conversation/list/container.js [hunk 2]
```diff
@@ -119,13 +119,13 @@ const mapDispatchToProps = (dispatch: Dispatch): DispatchProps => ({
     localMessageState?: Types.LocalMessageState,
     onShowEditor?: () => void,
     onPopupWillClose?: () => void,
-    targetNode?: ?HTMLElement
+    targetRect?: ?ClientRect
```
Comment (chrisnojima) [inline shared/chat/conversation/list/container.js]: why is this change required?

Comment (chrisnojima) [reply shared/chat/conversation/list/container.js]: so it'll work in native?

Comment (cjb) [reply shared/chat/conversation/list/container.js]: @chrisnojima it was causing V8 crashes! seems we can't put HTMLElements in routeState because (probably) when some part of our stack hits the circular webContents attr of the HTMLElement everything goes to 100% CPU.

Comment (chrisnojima) [reply shared/chat/conversation/list/container.js]: but is this cause you have routePrinting turned on?

Comment (cjb) [reply shared/chat/conversation/list/container.js]: It happened in a standard start-hot run.. even if it requires debugging on, would rather just keep it out of the routeState entirely, since many people do use debugging and when the crash happens there's no indication what the cause is.

**Comment by cjb** [comment]
Looks like circle has a big backlog, gonna merge anyway since the previous commit passed.
